### ISSUE TITLE
Wrong version text in v5.3.0

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#5778]
As reported in #5745 

https://cdn.jsdelivr.net/npm/@fullcalendar/common@5.3.0/main.js

```
/*!
FullCalendar v5.2.1
Docs & License: https://fullcalendar.io/
(c) 2020 Adam Shaw
*/
```

Also has this:

```
var version = '<%= version %>';
```

**Comment by arshaw** [comment]
Happens in a number of 5.3.0 packages. Problem with the build script most likely.

Does not affection functionality

**Comment by arshaw** [comment]
this has been released in v5.3.1!
https://github.com/fullcalendar/fullcalendar/releases/tag/v5.3.1
